tracer/src/Datadog.Trace/IAST/IastRequestContext.cs

```diff
@@ -5,28 +5,31 @@
 
 #nullable enable
 
+using System;
+
 namespace Datadog.Trace.Iast;
 
 internal class IastRequestContext
 {
+    private static int vulnerabilitiesPerRequest = Iast.Instance.Settings.VulnerabilitiesPerRequest;
```

Comment:
Is this setting allowed to change at any time?

Reply:
No. Anyway, it could be removed.